conftest.py

```diff
@@ -98,22 +98,22 @@ def pytest_addoption(parser):
         % (CACHE_OPT, S3_OPT),
     )
     parser.addoption(
-        S3_BUCKET_OPT,
+        S3_PATH_OPT,
         type=str,
         help="Url to s3 bucket with optional key. Example: s3://bucket_name/key/to/tests/",
         default=PYTEST_S3_PROVIDER_BASE_ROOT,
     )
-    parser.addoption(
-        FULL_BENCHMARK_OPT,
-        action="store_true",
-        help="Some benchmarks take a long time to run and by default should be skipped. This option enables them.",
-    )
     parser.addoption(
         KEEP_STORAGE_OPT,
         action="store_true",
         help="All storage providers/datasets will have their pytest data wiped. \
                 Use this option to keep the data after the test run.",
     )
+    parser.addoption(
+        FULL_BENCHMARK_OPT,
+        action="store_true",
+        help="Some benchmarks take a long time to run and by default should be skipped. This option enables them.",
```

Comment:
Is there any way to output these help texts from a script so we don't have to maintain two copies (one here and one in CONTRIBUTORS.md)?

That way we could just say:
> run `some_command` to view all available pytest options

Reply:
oh yeah i think we can do `pytest -h`

Reply:
the only problem is the custom options aren't isolatable in `pytest -h`. we could probably just link them to the line in the code responsible for defining all of them